### DISCUSSION TITLE
Support for `place-items: center`

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#2102] [Ideas]
Tailwind has `items-center` and `justify-center`. Now there is a new CSS feature that combine them.

```html
div {
  place-items: center;
}
```

It has good support: https://1linelayouts.glitch.me/

So it would be a nice addition to Tailwind as well. What is a good class name for it? Maybe just `center` because it center both x and y?

**Comment by adamwathan** [comment]
Existing discussion here 👍🏻

https://github.com/tailwindlabs/tailwindcss/issues/1517
